addons/web_editor/static/src/js/editor/odoo-editor/src/OdooEditor.js

```diff
@@ -692,7 +692,9 @@ export class OdooEditor extends EventTarget {
     }
 
     resetContent(value = '<p><br></p>') {
-        this.editable.innerHTML = value;
+        if (value) {
```

Comment:
Since `value` has a default value, this is basically only true if `value` is an empty string. According to @Goaman we want to reset the content in that case as well, but it should indeed never be an empty value so I believe we should use the default value.

In short, I think
```js
value = value.length ? value : '<p><br></p>';
```
would be better than `if (value)`.

Reply:
Hello, @Zinston 
Thanks for the suggestion above.I tried your solution and i also tried this kind of solution before, But in both the cases i met with this ( https://runbot.odoo.com/runbot/batch/972972/build/24239305 ) error.

Reply:
That doesn't look like it has anything to do with your PR. Have you tried rebasing your branch and pushing again?

Reply:
@dmo-odoo fyi

Reply:
@Zinston Yeah, i have tried everything, yet the problem stays.

Reply:
Damn... Indeed @sebgeelen has the same issue in his PR: https://github.com/odoo/odoo/pull/113530 and I believe in both cases the changes can have an influence on `isDirty` so I'm guessing that's the link. We'll see when we identify the problem in his PR, if we can apply the solution here as well. I'll keep you posted.
fyi @dmo-odoo